spark/src/test/scala/geotrellis/spark/io/index/zcurve/ZSpaceTimeKeyIndexSpec.scala

```diff
@@ -93,7 +93,7 @@ class ZSpaceTimeKeySpec extends FunSpec with Matchers{
     }
 
     it("generates indexes by string pattern") {
-      val pattern = "YM" //by months
+      val pattern = "YMM" //by months
```

Comment:
Not sure I follow the fix here. Was it only using a single digit of the month and if so which one?